ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/metrics/LogOnlyObserver.scala

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.benchtool.metrics
+
+import org.slf4j.LoggerFactory
+
+// TODO: add metrics
+class LogOnlyObserver[T] extends ManagedStreamObserver[T] {
+  private val logger = LoggerFactory.getLogger(getClass)
```

Comment:
I would suggest passing the logger as constructor argument. In this manner the messages will be scoped in the logfile to the same logger name as the message saying "Started fetching transactions"

Reply:
Good point! Thank you.